lib/generators/exception_hunter/install/install_generator.rb

```diff
@@ -15,7 +15,8 @@ def copy_initializer
 
     def setup_routes
       if options[:users]
-        inject_into_file 'config/routes.rb', "\n  ExceptionHunter.routes(self)", after: /devise_for .*/
+        inject_into_file 'config/routes.rb', "\n  ExceptionHunter.routes(self)",
+                         after: /devise_for :#{plural_table_name}.*/
```

Comment:
Where does this variable come from? Maybe we can make it a bit more explicit as to what it is.